packages/cli/src/commands/bundle/saveAssets.js

```diff
@@ -14,14 +14,19 @@ import fs from 'fs';
 import filterPlatformAssetScales from './filterPlatformAssetScales';
 import getAssetDestPathAndroid from './getAssetDestPathAndroid';
 import getAssetDestPathIOS from './getAssetDestPathIOS';
-import {logger} from '@react-native-community/cli-tools';
+import {logger, CLIError} from '@react-native-community/cli-tools';
 
 function saveAssets(assets, platform, assetsDest) {
   if (!assetsDest) {
     logger.warn('Assets destination folder is not set, skipping...');
     return Promise.resolve();
   }
 
+  if (!fs.existsSync(assetsDest)) {
+    let error = `The specified assets destination folder "${assetsDest}" does not exist.`;
+    throw new CLIError(error);
```

Comment:
Is `let error` needed here on line 26?

Reply:
No, I don't think so. I will remove this.